cpp/modmesh/view/RPythonConsoleDockWidget.cpp

```diff
@@ -154,7 +155,9 @@ void RPythonConsoleDockWidget::printCommandHistory()
     QTextCursor cursor = m_history_edit->textCursor();
     cursor.movePosition(QTextCursor::End);
     m_history_edit->setTextCursor(cursor);
-    m_history_edit->insertHtml(m_commandHtml + m_command_edit->toPlainText() + m_endHtml);
+    m_history_edit->insertPlainText(QString::fromStdString(
+        Formatter() << "[" << m_last_command_serial << "] "
```

Comment:
The prompt will look like:

```
[1] print(True)
True
```

Reply:
suggestion, maybe one more blank line between two commands.
```
[1] print(True)
True

[2] print(True)
True
```

Reply:
Good idea.  Patch is coming.